hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -355,6 +355,7 @@
       uvm_test_seq: chip_sw_base_vseq
       sw_images: ["sw/device/tests/rstmgr_sw_req_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=8_000_000"]
```

Comment:
I very recently changed the default timeout to 12ms, so this should not be needed: 
https://github.com/lowRISC/opentitan/blob/8920b8c692b1aa305099e97914621f7cb9a76552/hw/top_earlgrey/dv/env/chip_env_cfg.sv#L80

Reply:
awesome, removed